test/crawler_web/controllers/api/v1/user_registration_controller_test.exs

```diff
@@ -0,0 +1,84 @@
+defmodule CrawlerWeb.Api.V1.UserRegistrationControllerTest do
+  use CrawlerWeb.ConnCase, async: true
+
+  alias Crawler.Account.Guardian
+
+  describe "POST create/2" do
+    test "given valid user credentials, returns access token", %{conn: conn} do
+      email = "new_email@gmail.com"
+
+      conn =
+        post(conn, Routes.api_v1_user_registration_path(conn, :create), %{
+          email: email,
+          password: "12345678"
+        })
+
+      assert %{
+               "data" => %{
+                 "attributes" => %{
+                   "email" => ^email,
+                   "token" => _
+                 },
+                 "id" => _,
+                 "relationships" => %{},
+                 "type" => "user_registration"
+               },
+               "included" => []
+             } = json_response(conn, 200)
+    end
+
+    test "given invalid user credentials, returns error response", %{conn: conn} do
+      conn = post(conn, Routes.api_v1_user_registration_path(conn, :create), %{})
+
+      assert json_response(conn, 422) == %{
+               "errors" => [
+                 %{
+                   "code" => "unprocessable_entity",
+                   "detail" => gettext("Email or password is missing")
+                 }
+               ]
+             }
+    end
+
+    test "given existing user credentials, returns error response", %{conn: conn} do
```

Comment:
should we test for the invalid password case 🤔 like `password is less than 6 characters` 🤔